src/main/java/net/minecraftforge/event/BiomeLoadingEvent.java

```diff
@@ -0,0 +1,82 @@
+package net.minecraftforge.event;
```

Comment:
This file also needs its license header.

Reply:
Indeed